roboteam_ai/src/ApplicationManager.cpp

```diff
@@ -114,7 +114,7 @@ void ApplicationManager::runOneLoopCycle() {
                 ai::robotDealer::RobotDealer::setKeeperID(ai::world::world->getUs().at(0).id);
             }
             keeperTree = BTFactory::getKeeperTree();
-            if (keeperTree) {
+            if (keeperTree && rtt::ai::robotDealer::RobotDealer::keeperExistsInWorld()) {
```

Comment:
We could have a get keeper back in vision tactic that we tick if the first one is true or in any case not ignore the fact that we used to have a keeper and now we dont